bucket/kombucha.json

```diff
@@ -0,0 +1,13 @@
+{
+    "homepage": "https://github.com/lptstr/kombucha",
+    "version": "0.1.0",
+    "url": "https://codeload.github.com/lptstr/kombucha/zip/v0.1.0#/dl.7z",
```

Comment:
Should this be .zip? Or does 7zip not care it’s misnamed?

Reply:
From what I've seen of other manifests, 7zip doesn't seem to care whether its extracting ZIP files or 7Z files. I also know that Scoop uses 7zip to extract itself during updates.

Reply:
Cool. Shoulda checked that myself.